components/match2/commons/match_group_input_util.lua

```diff
@@ -1243,7 +1243,7 @@ end
 
 ---@class FfaMatchParserInterface
 ---@field extractMaps fun(match: table, opponents: table[], mapProps: any?): table[]
----@field parseSettings fun(match: table): table
+---@field parseSettings fun(match: table, opponentCount: integer): table
```

Comment:
don't we need to adjust more customs for this?

Reply:
Yes